pallets/reservable-currency/src/lib.rs

```diff
@@ -91,11 +94,15 @@ decl_module! {
 		) -> DispatchResult {
 			let _ = ensure_signed(origin)?; // dangerous because can be called with any signature (so dont do this in practice ever!)
 
+                        // Unreserved contains our overdraft, if collateral is bigger than
+                        // to_punish's reserved balance, what's left is put in unreserved.
 			let unreserved = T::Currency::unreserve(&to_punish, collateral);
```

Comment:
That's correct. Probably better to change the name of the variable to something like `overdraft` or `shortfall` or something.

Reply:
I agree! Overdraft it is!